templates/systemd-gitea-backup.service.conf.j2

```diff
@@ -0,0 +1,16 @@
+{{ ansible_managed | comment }}
+
+[Unit]
+Description={{ gitea_fork }} backup
+StartLimitIntervalSec=0
+Documentation=https://docs.gitea.com/administration/backup-and-restore
+Documentation=https://forgejo.org/docs/latest/admin/upgrade/#backup
+
+[Service]
+Type=oneshot
+User=gitea
+ExecStart=/usr/local/bin/{{ gitea_fork }} dump --config /etc/gitea/gitea.ini
+WorkingDirectory={{ backupdir }}
```

Comment:
where is the backupdir variable defined?

Reply:
Yes, it works as is. Even if as often for db stuff, it is recommended to stop service to ensure consistency.